cpp/src/parquet/encryption/internal_file_decryptor.cc

```diff
@@ -64,16 +64,6 @@ InternalFileDecryptor::InternalFileDecryptor(FileDecryptionProperties* propertie
   properties_->set_utilized();
 }
 
-void InternalFileDecryptor::WipeOutDecryptionKeys() {
-  std::lock_guard<std::mutex> lock(mutex_);
-  properties_->WipeOutDecryptionKeys();
-  for (auto const& i : all_decryptors_) {
```

Comment:
I think being able to wipe out the decryptors is the only reason the `all_decryptors_` and `mutex_` members exist, so those should be able to be removed.

Reply:
thanks, done